scripts/tools/api.py

```diff
@@ -69,7 +69,7 @@ def create_service(user_id, service_name):
         "username": "caicloud",
         "repository": {
             #"url": "/home/superxi/gopath/src/github.com/caicloud/console-web",
-            "url": "https://github.com/zoumo/go_test.git",
+            "url": "https://github.com/zz-bmj/go-demo",
```

Comment:
The new repo is just for your test? Whether the old repo still can work? If it still can work, no need to change it.

Reply:
the old repo name has been changed, so I fork it from "zoumo" and then change it with my repo.